README.md

```diff
@@ -2,7 +2,7 @@ Terraform
 =========
 
 - Website: http://www.terraform.io
-- IRC: `#terraform-tool` on Freenode
+- [![Gitter chat](https://badges.gitter.im/terraform-tool/Lobby.png)](https://gitter.im/terraform-tool/Lobby)
```

Comment:
Should this have been `https://gitter.im/hashicorp-terraform/Lobby` instead? The URL here doesn't seem to work.